src/applications/mhv-landing-page/app-entry.jsx

```diff
@@ -2,11 +2,26 @@ import '@department-of-veterans-affairs/platform-polyfills';
 import './sass/mhv-landing-page.scss';
 import '~/platform/mhv/secondary-nav/sass/mhv-sec-nav.scss';
 import { startAppFromRouter as startApp } from '@department-of-veterans-affairs/platform-startup/exports';
+import { initializeDatadogRum } from './utilities/datadogRum';
 
 import routes from './routes';
 import reducer from './reducers';
 import manifest from './manifest.json';
 
+initializeDatadogRum({
+  applicationId: '1f81f762-c3fc-48c1-89d5-09d9236e340d',
+  clientToken: 'pub3e48a5b97661792510e69581b3b272d1',
+  site: 'ddog-gov.com',
+  service: 'mhv-on-va.gov',
+  sessionSampleRate: 100,
+  sessionReplaySampleRate: 10,
+  trackInteractions: true,
+  trackUserInteractions: true,
```

Comment:
vets-website is currently using [`@datadog/browser-rum` at version 4.50.1](https://github.com/department-of-veterans-affairs/vets-website/blob/bad0453/yarn.lock#L2812)

It looks as if the `trackInteractions` option was changed to `trackUserInteractions` in versions > 4.30.0. ([link](https://docs.datadoghq.com/real_user_monitoring/browser/setup/#npm), expand 'before v5.0.0' and 'before v4.30.0' sections.) I'm thinking the `trackInteractions` property can be removed.

Reply:
Good call